examples/components/time-picker/default.example

```diff
@@ -0,0 +1,8 @@
+class Example extends React.Component {
```

Comment:
this example should be improved to store in `this.state` the `value` returned by `onChange` and pass it to `TimePicker`